### DISCUSSION TITLE
posts/applying-1password-open-source-plan

### DISCUSSION THREAD

**Discussion by giscus** [at-wr/blog#2] [Blog Posts Comments]
# posts/applying-1password-open-source-plan

当今，大部分平台已逐步从“一次性买断”过渡到“订阅制”，1Password 8 也是如此。科技行业日新月异，公司为了维持经营、继续发展做出此番举动也并非不可理解。但是只要你申请通过了 1Password for Open Source Project，你就可以免费获得永久的 1Password for Teams 订阅...

https://wrye.dev/posts/applying-1password-open-source-plan

**Comment by AmosHuKe** [comment]
才发现有这样的许可证申请~

**Reply by at-wr** [reply]
这个好像 2018 年就有了，不过我也是去年才知道的~ 
[这个](https://github.com/1Password/1password-teams-open-source/pull/2) (好像) 是第一个申请的 PR 🤔

**Comment by LO0215** [comment]
很有意思的博客，请问下这个password的安全性如何保证呢？我现在是对于这种密码管理器持有怀疑态度，比如google，比如apple？

**Comment by haotzops** [comment]
刚跟别人合租一个月 :sob:
